serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/actions/NavigateToPage.java

```diff
@@ -0,0 +1,57 @@
+package net.serenitybdd.screenplay.webtests.actions;
+
+import net.serenitybdd.screenplay.Actor;
+import net.serenitybdd.screenplay.Task;
+import net.serenitybdd.screenplay.actions.Click;
+import net.serenitybdd.screenplay.targets.Target;
+import net.serenitybdd.screenplay.webtests.model.Category;
+import net.serenitybdd.screenplay.webtests.model.SubCategory;
+import net.serenitybdd.core.pages.PageObject;
+import net.serenitybdd.screenplay.webtests.pages.JLNavigationBar;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+
+import java.time.Duration;
+
+import static net.serenitybdd.screenplay.Tasks.instrumented;
+
+
+public class NavigateToPage extends PageObject implements Task {
+
+    private final Category menu;
+    SubCategory item;
+
+    public NavigateToPage(Category menu){
+        this.menu = menu;
+    }
+
+    public static NavigateToPage withMenu(Category menu){
+        return instrumented(NavigateToPage.class, menu);
+    }
+
+    public NavigateToPage selectItem(SubCategory item){
+        this.item = item;
+        return this;
+    }
+
+    @Override
+    public <T extends Actor> void performAs(T actor) {
+        waitABit(10000);
```

Comment:
A task is performed by an actor, and a UI task typically uses the actor's driver instance. Here you are mixing imperative/Page Object-style code with more functional-style Screenplay tasks, which doesn't really work well. If you need to wait for an element to appear, just get the actor to do the waiting.

Reply:
Hi @wakaleo I'm sorry for my mistakes to create this pull request by accidentally. However, I'll appreciate for your kind feedback. Could you please me to close this one? Appreciate and thank you!